audio-worklet/src/param-controller.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+class ParamController {
+  /**
+   * Event handler and layout for audio parameters with numeric input values
+   * @param  {String} parentId id for parent element
+   * @param  {Function} event callback to trigger on input
```

Comment:
I am confused by this. `event` is JS code should mean `event` object. Rename the variable and please be more descriptive about what this does. Perhaps `onChangeCallback` or `notifyCallback`?